packages/graphql-codegen-supermassive-schema-extraction-plugin/CHANGELOG.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "@graphitation/graphql-codegen-supermassive-typed-document-node-plugin",
+  "entries": [
+    {
+      "date": "Mon, 20 Jun 2022 12:54:16 GMT",
+      "tag": "@graphitation/graphql-codegen-supermassive-typed-document-node-plugin_v0.1.2",
+      "version": "0.1.2",
+      "comments": {
+        "patch": [
+          {
+            "author": "jakubvejr@microsoft.com",
+            "package": "@graphitation/graphql-codegen-supermassive-typed-document-node-plugin",
+            "commit": "68f1b925f86d314f156b9b9e42514e329f036046",
+            "comment": "version bump - npmignore modified"
+          }
+        ]
+      }
+    },
+    {
+      "date": "Thu, 16 Jun 2022 21:57:11 GMT",
+      "tag": "@graphitation/graphql-codegen-supermassive-typed-document-node-plugin_v0.1.1",
+      "version": "0.1.1",
+      "comments": {
+        "patch": [
+          {
+            "author": "eloy.de.enige@gmail.com",
+            "package": "@graphitation/graphql-codegen-supermassive-typed-document-node-plugin",
+            "commit": "967f207d3a5e69bcfb4b2f146cec67179a453e48",
+            "comment": "No-op bump"
+          }
+        ]
+      }
+    },
+    {
+      "date": "Mon, 13 Jun 2022 15:37:16 GMT",
+      "tag": "@graphitation/graphql-codegen-supermassive-typed-document-node-plugin_v0.1.0",
+      "version": "0.1.0",
+      "comments": {
+        "none": [
+          {
+            "author": "beachball",
+            "package": "@graphitation/graphql-codegen-supermassive-typed-document-node-plugin",
+            "comment": "Bump @graphitation/supermassive to v0.8.3",
+            "commit": "a0a12b2193c85d419cf018560adce1370afa4121"
+          }
+        ]
+      }
+    },
+    {
+      "date": "Fri, 10 Jun 2022 14:09:50 GMT",
+      "tag": "@graphitation/graphql-codegen-supermassive-typed-document-node-plugin_v0.1.0",
+      "version": "0.1.0",
+      "comments": {
+        "minor": [
+          {
+            "author": "jakubvejr@microsoft.com",
+            "package": "@graphitation/graphql-codegen-supermassive-typed-document-node-plugin",
+            "commit": "9985dd0fb3e8b80807b3118e6b5f2d20df8397b3",
+            "comment": "Supermassive typed document node plugin added"
+          },
+          {
+            "author": "beachball",
+            "package": "@graphitation/graphql-codegen-supermassive-typed-document-node-plugin",
+            "comment": "Bump @graphitation/supermassive to v0.8.3",
+            "commit": "2f8bfe127192958c94261fd1e6ba998247204a31"
+          }
+        ]
+      }
+    }
+  ]
+}
```

Comment:
Did you copy this file from the other plugin?

Reply:
Yes, fixed